test/unit/foreman/renderer/snapshots.yaml

```diff
@@ -1,71 +1,264 @@
+  - 
+    name: user_data/kickstart_default_user_data.erb
+    os:
+      - ["CentOS", "Redhat", 6, 0]
+      - ["CentOS", "Redhat", 7, 0]
+      - ["CentOS", "Redhat", 8, 0]
+  - 
+    name: provision/atomic_kickstart_default.erb
+    os:
+      - ["CentOS", "Redhat", 6, 0]
+      - ["CentOS", "Redhat", 7, 0]
+      - ["CentOS", "Redhat", 8, 0]
+# preseed
+  - 
+    name: provision/preseed_default.erb
+    os:
+      - ["Debian", "Debian", 8, 0]
+      - ["Ubuntu", "Debian", 16, 04]
+  - 
+    name: finish/preseed_default_finish.erb
+    os:
+      - ["Debian", "Debian", 8, 0]
+      - ["Ubuntu", "Debian", 16, 04]
+  - 
+    name: PXELinux/preseed_default_pxelinux.erb
+    os:
+      - ["Debian", "Debian", 8, 0]
+      - ["Ubuntu", "Debian", 16, 04]
+  - 
+    name: PXEGrub2/preseed_default_pxegrub2.erb
+    os:
+      - ["Debian", "Debian", 8, 0]
+      - ["Ubuntu", "Debian", 16, 04]
+  - 
+    name: user_data/preseed_default_user_data.erb
+    os:
+      - ["Debian", "Debian", 8, 0]
+      - ["Ubuntu", "Debian", 16, 04]
+# suse
+  - 
+    name: provision/autoyast_default.erb
+    os:
+      - ["openSUSE", "Suse", 13, 0]
+      - ["openSUSE", "Suse", 14, 0]
+      - ["openSUSE", "Suse", 15, 0]
+  - 
+    name: provision/autoyast_sles_default.erb
+    os:
+      - ["openSUSE", "Suse", 13, 0]
+      - ["openSUSE", "Suse", 14, 0]
+      - ["openSUSE", "Suse", 15, 0]
+  - 
+    name: PXELinux/autoyast_default_pxelinux.erb
+    os:
+      - ["openSUSE", "Suse", 13, 0]
+      - ["openSUSE", "Suse", 14, 0]
+      - ["openSUSE", "Suse", 15, 0]
+  - 
+    name: user_data/autoyast_default_user_data.erb
+    os:
+      - ["openSUSE", "Suse", 13, 0]
+      - ["openSUSE", "Suse", 14, 0]
+      - ["openSUSE", "Suse", 15, 0]
+# coreos
+  - 
+    name: provision/coreos_provision.erb
+    os:
+      - ["CoreOS", "Coreos", 0, 0]
+  - 
+    name: PXELinux/coreos_pxelinux.erb
+    os:
+      - ["CoreOS", "Coreos", 0, 0]
+# windows
+  - 
+    name: PXELinux/waik_default_pxelinux.erb
+    os:
+      - ["Windows", "Windows", 10, 0]
+# TODO uncomment the new Windows templates for 1.25
+#  - 
+#    name: partition_tables_templates/windows_default.erb
+#    os:
+#      - ["Windows", "Windows", 10, 0]
+#  - 
+#    name: provisioning_templates/PXELinux/windows_default_pxelinux.erb
+#    os:
+#      - ["Windows", "Windows", 10, 0]
+#  - 
+#    name: provisioning_templates/finish/windows_default_finish.erb
+#    os:
+#      - ["Windows", "Windows", 10, 0]
+#  - 
+#    name: provisioning_templates/iPXE/windows_default_ipxe.erb
+#    os:
+#      - ["Windows", "Windows", 10, 0]
+#  - 
+#    name: provisioning_templates/provision/windows_default_provisioning.erb
+#    os:
+#      - ["Windows", "Windows", 10, 0]
+#  - 
+#    name: provisioning_templates/script/windows_default_script.erb
+#    os:
+#      - ["Windows", "Windows", 10, 0]
+#  - 
+#    name: provisioning_templates/snippet/windows_networking_snippet.erb
+#    os:
+#      - ["Windows", "Windows", 10, 0]
```

Comment:
Are you fine with such a comment or do you want me to create a ticket? @tbrisker

Reply:
https://projects.theforeman.org/issues/28153